var/spack/repos/builtin/packages/py-antlr4-python3-runtime/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyAntlr4Python3Runtime(PythonPackage):
```

Comment:
@adamjstewart Do you support naming this `py-antlr4-python3-runtime` because that's what it's called in PiPI?  Or do we simplify to the easier-to-understand `py-antlr4`?

Reply:
I prefer the current name. If that's what the developers call it on PyPI, we should honor that choice.